src/miner.cpp

```diff
@@ -925,14 +918,16 @@ void StakeMiner(CWallet *pwallet)
             else
             {
                 msMiningErrors = "Stake block rejected";
-                if (fDebug) printf("Stake block rejected \r\n");
+                printf("Stake block rejected \r\n");
                 MilliSleep(1000);
             }
             SetThreadPriority(THREAD_PRIORITY_LOWEST);
```

Comment:
ok
But i doubt it does anything since only root can ever raise priority of thread.